src/avatar/styles.scss

```diff
@@ -0,0 +1,42 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+@use './mixins.scss' as mixins;
+@use '../internal/styles' as styles;
+@use '../internal/styles/tokens' as awsui;
+@use '../internal/hooks/focus-visible' as focus-visible;
+
+$avatar-size: 28px;
+
+.root {
+  @include styles.styles-reset;
+
+  display: flex;
+  color: awsui.$color-text-avatar;
+  block-size: $avatar-size;
+  inline-size: $avatar-size;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  overflow: hidden;
+  @include mixins.border-radius-avatar;
+
+  &.avatar-color-default {
+    background-color: awsui.$color-background-avatar-default;
+  }
+
+  &.avatar-color-gen-ai {
+    background: awsui.$color-background-avatar-gen-ai;
+  }
+
+  &.initials {
+    font-family: awsui.$font-family-base;
+    font-size: 12px;
+    line-height: 16px;
+  }
+
+  @include focus-visible.when-visible {
```

Comment:
We should be able to use the existing `focus-highlight` mixin to draw the focus outline